js/views/playingScreen.js

```diff
@@ -69,6 +69,17 @@ var PlayingScreen = React.createClass({
     // for mobile, touch is handled in handleTouchEnd
   },
 
+  handleKeyPress: function(event) {
+    // show control bar on tab key navigation
+    if(event.which === 9 || event.keyCode === 9) {
+      this.showControlBar();
+      this.props.controller.startHideControlBarTimer();
+      if(this.props.controller.showControlBarByTab) {
```

Comment:
Is this for unit tests, or are there cases in which this function is not defined?